Assets/SEE/Utils/Permuter.cs

```diff
@@ -0,0 +1,61 @@
+﻿using System.Collections.Generic;
+
+namespace SEE.Utils
+{
+    /// <summary>
+    /// Allows to permute a list of values.
+    /// </summary>
+    /// <typeparam name="T">the type of the values</typeparam>
+    internal static class Permuter<T>
```

Comment:
The permuter is a re-implementation of an already existing extension method within SEE, as we needed to calculate permutations before. You can simply call `values.Permutations()` for the same effect (implemented in `Utils/CollectionExtensions.cs`).

(I think this isn't the first time that the problem of "someone implements something that already exists somewhere else within SEE" has occurred, but I don't really know what to do about it 😅)

Reply:
I removed `Permuter` again.

Type-4 clones are a general problem. In most cases impossible to detect automatically. Their root causes are often unawareness and lack of communication. But at least this review process has helped.